packages/eas-cli/src/commands/build/index.ts

```diff
@@ -60,51 +58,47 @@ export default class Build extends Command {
 
   async run(): Promise<void> {
     const { flags } = this.parse(Build);
```

Comment:
removed the `try` `finally` blocks here as we weren't actually catching any errors and the flush got moved into the base class